c7n/filters/related.py

```diff
@@ -70,9 +70,17 @@ def process_resource(self, resource, related):
         model = self.manager.get_model()
         op = self.data.get('operator', 'or')
         found = []
+
         if self.data.get('match-resource') is True:
             self.data['value'] = self.get_resource_value(
                 self.data['key'], resource)
+
+        if self.data.get('value_type') == 'resource_count':
```

Comment:
any reason why not to add this directly to value filter ?

Reply:
I'm not sure I understand what you mean.

Do you mean I should be using something from `filters/core.py` ?  There is some similar code in there, but it was not being used in `filters/related.py` ...

Reply:
never mind, ic why this was done here, thanks